### PR TITLE
supports pre allocation of iptags and reverse iptags

### PR DESCRIPTION
**Pull request by alan-stokes**
this supports the pre allocation of iptags and reverse iptags. this should (i think) have been done during LPG stuff, but I only picked it up when starting the integration of the 15 speed up data extractor.

tested with spike io.
needs a FEC equiv brnach to be merged as well
https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/226

togehter complete issue 
https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/issues/209

### CODE REVIEW

**Comment by dkfellows** [comment]
“Preallocated” is a single word in normal English.

**Comment by Christian-B** [comment]
On hold awaiting a similar named pull request in SpiNNFrontEndCommon

File: pacman/model/graphs/abstract_graph.py [hunk 1]
```diff
@@ -220,12 +220,3 @@ def get_outgoing_edge_partition_starting_at_vertex(
         :rtype:\
             :py:class:`pacman.model.graphs.abstract_outgoing_edge_partition.AbstractOutgoingEdgePartition`
         """
-
-    @abstractmethod
-    def get_outgoing_edge_partitions_with_traffic_type(self, traffic_type):
-        """ Get the outgoing edge partitions with a given traffic type
-
```
Comment (dkfellows) [inline pacman/model/graphs/abstract_graph.py]: I'm curious as to why this is being removed as part of this changeset.

Comment (alan-stokes) [reply pacman/model/graphs/abstract_graph.py]: nnone uses it at all. dead code. so clean up

**Pull request merged**